libs/components/src/next/form/datepicker/stories/vl-datepicker.stories-arg.ts

```diff
@@ -0,0 +1,149 @@
+import { CATEGORIES, defaultArgs, defaultArgTypes, TYPES } from '@domg-wc/common-storybook';
+import { ArgTypes } from '@storybook/web-components';
+import { formControlArgs, formControlArgTypes } from '../../form-control/stories/form-control.stories-arg';
+import { DatepickerDefaults } from '../vl-datepicker.component';
+
+export const datepickerArgs = {
+    ...defaultArgs,
+    ...formControlArgs,
+    ...DatepickerDefaults,
+};
+
+export const datepickerArgTypes: ArgTypes = {
+    ...defaultArgTypes(true),
+    ...formControlArgTypes,
+    block: {
+        name: 'block',
+        description: 'Duidt aan dat de component de volledige breedte van zijn parent mag innemen.',
+        table: {
+            type: { summary: TYPES.BOOLEAN },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: datepickerArgs.block },
+        },
+    },
+    readonly: {
+        name: 'readonly',
+        description: 'Duidt aan dat het veld enkel leesbaar is.',
+        table: {
+            type: { summary: TYPES.BOOLEAN },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: datepickerArgs.readonly },
+        },
+    },
+    value: {
+        name: 'value',
+        description: 'De waarde van het veld.',
+        table: {
+            type: { summary: TYPES.STRING },
+            category: CATEGORIES.ATTRIBUTES,
+            defaultValue: { summary: datepickerArgs.value },
+        },
+    },
+    type: {
+        name: 'type',
+        description: "Attribuut bepaalt het soort datepicker; range | time | date-time | '' (default)",
```

Comment:
Overal 'Attribuut ...' in de descriptions verwijderen aub.

Reply:
Done.